### ISSUE TITLE
Nice to have feature: open putc getc syscalls in CAT BIOS 

### ISSUE THREAD

**Issue by lennart-benschop** [breakintoprogram/cerberus-bbc-basic#3]
If you can create extra space in CAT BIOS, you could go for this.

Could make the OPENIN, OPENOUT, INPUT#, PRINT# commands in BASIC work. 
Plus of course the ability to load and save in plain ASCII, like Agon has for some time.

**Comment by breakintoprogram** [comment]
Yeah, I'm down to the last 2K and would love to get subdirectory support in as a higher priority.
